extensions/binding/org.eclipse.smarthome.binding.iota/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bindingId="iota" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+   <!-- IOTA Binding -->
+   <thing-type id="topic" extensible="text,number,percentage,onoff">
+      <supported-bridge-type-refs>
+          <bridge-type-ref id="bridge"/>
+      </supported-bridge-type-refs>
+      <label>Topic</label>
+      <description>This thing represents a topic</description>
+      <config-description>
+         <parameter name="root" type="text" required="true">
+            <label>Root Address</label>
+            <description>The root address from which item states should be retrieved from</description>
+         </parameter>
+         <parameter name="refresh" type="integer" min="5">
```

Comment:
Imho it'd be better if maybe we force the user to not leave this field blank, using the `required=true` parameter. This way the user won't be able to use the thing at all if not properly configured. Otherwise I can't see what default value would be good to use. A default root address (even one that actually maps to some message on the Tangle) would get erased from the Tangle eventually after a snapshot, so not of any use. Wdyt would be best?